src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/help.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+
+helps['monitor'] = """
+            type: group
+            short-summary: Commands to manage Azure Monitor service.
+            """
+# MANAGEMENT COMMANDS HELPS
+helps['monitor alert-rules'] = """
+            type: group
+            short-summary: Commands to manage alert rules.
```

Comment:
It might make sense to clarify "manage Monitor alert rules" etc. for these commands since the terms are pretty generic and could be confused for other things.